src/tests/electron/jest.config.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const path = require('path');
+var common = require('../jest.common.config');
+const rootDir = '../../../';
+const currentDir = '<rootDir>/src/tests/electron';
+
+module.exports = {
+    ...common,
+    displayName: 'electron tests',
+    moduleFileExtensions: ['ts', 'js'],
+    rootDir: rootDir,
+    roots: [currentDir],
+    reporters: ['default', ['jest-junit', { outputDirectory: '.', outputName: '<rootDir>/test-results/e2e/junit-e2e.xml' }]],
```

Comment:
Should probably use a separate results file

Reply:
Oops, meant to do that. Fixed now.